tests/parametric/test_process_discovery.py

```diff
@@ -9,7 +9,7 @@
 
 
 def find_dd_memfds(test_library, pid: int) -> list[str]:
-    rc, out = test_library.container_exec_run(f"find /proc/{pid}/fd -lname '/memfd:datadog-tracer-info*'")
+    rc, out = test_library.container_exec_run(f"find /proc/{pid}/fd -lname '/memfd:datadog-tracer-info-*'")
```

Comment:
added the `-` [to match agent matcher]([just realised the agent implementation](https://github.com/DataDog/datadog-agent/blob/main/pkg/discovery/tracermetadata/tracer_memfd.go#L17))